src/main/java/org/launchcode/water_garden_tour/controllers/AdminController.java

```diff
@@ -59,8 +60,8 @@ public String renderAddGarden(Model model) {
     }
 
     @PostMapping("/gardens/add")
-    public String addGarden(@ModelAttribute @Valid Garden newGarden, @RequestParam("file") MultipartFile file,
-                            Errors errors, Model model) throws IOException {
+    public String addGarden(@ModelAttribute @Valid Garden newGarden, Errors errors, @RequestParam("file") MultipartFile file,
```

Comment:
Reordering Errors errors was Heather's helpful discovery, you'll see this throughout the update controllers